manifests/manage/bzr.pp

```diff
@@ -0,0 +1,18 @@
+# @summary Manage the Bazaar source code manager package
+#
+# @param package_name
+#   name of package to manage
+#
+# @param package_ensure
+#   ensure state of the package resource
+#
+# @example simple inclue
```

Comment:
Looks like the typo was copied to other files as well
```suggestion
# @example simple include
```

Reply:
oops, fixed.